### ISSUE TITLE
Thunderball first two suggested numbers are too similar

### ISSUE THREAD

**Issue by markheydon** [markheydon/national-lottery-generator#84]
For example, the following are some Suggested Numbers, lines 1 and 2 are very similar and therefore probably not much use for the Thunderball game given that it is more about matching the Thunderball number plus other numbers.

Line 1: 03 - 12 - 26 - 31 - 38 ** 10
Line 2: 18 - 21 - 26 - 31 - 38 ** 10
Line 3: 02 - 14 - 16 - 22 - 38 ** 03

Suggest dropping the current line 2, making line 1 to be line 2 and making some new logic for a new line 1 as something like the following.

- Work out the most populate Thunderball number across all lines generated. 
- Using the first result found for line 1.

So given the above example numbers, the new suggested would be like the following.

Line 1: 01 - 03 - 10 - 29 - 38 ** 14
Line 2: 03 - 12 - 26 - 31 - 38 ** 10
Line 3: 02 - 14 - 16 - 22 - 38 ** 03

Basically just want *something* so the first thing lines generated aren't so similar.
